app/src/components/fields/fieldErrorHint/fieldErrorHint.scss

```diff
@@ -11,6 +11,7 @@
   left: 0;
   width: 100%;
   height: 0;
+  top: 0;
```

Comment:
This change breaks other fields with hints. Check login page for examples.

Reply:
fixed